example/lib/sample_circular_page.dart

```diff
@@ -7,195 +7,218 @@ class SampleCircularPage extends StatefulWidget {
-            center: Text(
-              "70.0%",
-              style: TextStyle(fontWeight: FontWeight.bold, fontSize: 20.0),
+            Padding(
+              padding: EdgeInsets.all(15.0),
+              child: CircularPercentIndicator(
+                radius: 60.0,
+                lineWidth: 5.0,
+                percent: 1.0,
+                center: Text("100%"),
+                progressColor: Colors.green,
+              ),
             ),
-            footer: Text(
-              "Sales this week",
-              style: TextStyle(fontWeight: FontWeight.bold, fontSize: 17.0),
+            Container(
+              padding: EdgeInsets.all(15.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  CircularPercentIndicator(
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    percent: 0.10,
+                    center: Text("10%"),
+                    progressColor: Colors.red,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  CircularPercentIndicator(
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    backgroundWidth: 1.0,
+                    percent: 0.2,
+                    animation: true,
+                    center: Text("20%"),
+                    progressColor: Colors.orangeAccent,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  CircularPercentIndicator(
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    percent: 0.30,
+                    center: Text("30%"),
+                    progressColor: Colors.orange,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  CircularPercentIndicator(
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    backgroundWidth: 8,
+                    animation: true,
+                    animationDuration: 200,
+                    percent: 0.60,
+                    center: Text("60%"),
+                    progressColor: Colors.yellow,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  CircularPercentIndicator(
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    percent: 0.90,
+                    center: Text("90%"),
+                    progressColor: Colors.green,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  // CircularPercentIndicator(
```

Comment:
why this was commented?

Reply:
I forgot to delete the comment. There was an overflow error and I did comment on it temporarily but forgot